frontend/src/components/login_page/login.css

```diff
@@ -0,0 +1,167 @@
+.login-page{
+    background-color: var(--black);
+    font-family: 'Gill Sans', sans-serif;
+    display: flex;
+    justify-content: space-between;
+    height: 100vh;
+    width: auto;
+}
+
+.form-side {
+    background-color: var(--tan);
+    width: 50%;
+    margin: 3vh 1.5vh 3vh 3vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    line-height: 1;
+}
+
+.image-side {
+    width: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 3vh 3vh 3vh 1.5vh;
+}
+
+#logo_symbol{
```

Comment:
we generally want to avoid having to use IDs in CSS

https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4